services/cirrus/cirrus.test.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
```

Comment:
Make sure to name this file `cirrus.tester.js` so that it will be properly picked up by our test runner